pkg/maroto/col.go

```diff
@@ -0,0 +1,10 @@
+package maroto
+
+import "github.com/jung-kurt/gofpdf"
+
+type Col struct {
+}
+
+func (c Col) draw(fpdf *gofpdf.Fpdf) {
```

Comment:
Maybe we should think to use something like:

```go
type Col func(fpdf *gofpdf.Fpdf)
```

So, the usage will be simpler, like:
```go
m.Col(fpdf)
```